.gitlab/ci/.gitlab-ci.global.yml

```
@@ -245,12 +245,15 @@
       git config core.fileMode false  # used to tell git not to identify permission changes on files as changes
       chmod -R 777 ./Packs  # required for the lint, we use the permissions of the file when running pytest within the containers
       echo "the Packs changes between upload commit ${LAST_UPLOAD_COMMIT} to master commit ${CI_COMMIT_SHA} is:"
-      git status -- Packs  # show the differences between the upload commit to the master branch for the Packs folder
+      git status -- Packs  # show the differences between the upload commit to the master branch for the Packs folder before commit
+      git commit -am "updated branch to upload commit ${LAST_UPLOAD_COMMIT}"
+      git status -- Packs # show the differences between the upload commit to the master branch for the Packs folder after commit
       echo "The Packs folder is in the state of commit $LAST_UPLOAD_COMMIT"
-    else
-      echo "not checking out to the latest upload commit $LAST_UPLOAD_COMMIT because the build is not content nightly"
+    elif [[ "${NIGHTLY}" != true ]]; then
+      echo "not checking out to the latest upload commit $LAST_UPLOAD_COMMIT because the build is not content nightly."
+    elif [[ ${LAST_UPLOAD_COMMIT} == ${CI_COMMIT_SHA} ]]; then
+      echo "not checking out to the latest upload commit because the ${LAST_UPLOAD_COMMIT} and ${CI_COMMIT_SHA} are equal."
```

```suggestion
      echo "not checking out to the latest upload commit because the \$LAST_UPLOAD_COMMIT and \$CI_COMMIT_SHA are equal to ${CI_COMMIT_SHA}."
```
i'm printing the variables names, and then i'm printing the value once, since it's equal.